features/support/page_objects/personal_details_page.rb

```diff
@@ -43,19 +43,26 @@ def valid_dob
     content.application_year_date_of_birth.set '1986'
   end
 
+  def valid_dob_over
```

Comment:
I think this method will need to be inside the  "# rubocop:disable Metrics/AbcSize" section